corehq/apps/export/models/new.py

```diff
@@ -254,6 +255,9 @@ class ExportDataSchema(Document):
     form xmlns or case type. It contains a list of ExportGroupSchema.
     """
     group_schemas = SchemaListProperty(ExportGroupSchema)
+
+    # A map of app_id to app_version. Represents the last time it saw an app and at what version
+    last_updated = DictProperty()
```

Comment:
`last_updated` makes me think its going to be a date. Maybe `last_app_versions` or something?